flake.nix

```diff
@@ -126,15 +137,23 @@
               });
 
               rustfmt = craneLib.cargoFmt (commonArgs // { inherit cargoArtifacts; });
-            };
+            }
+          );
 
           stubCrane = buildRustApp {
             src = craneLib.cleanCargoSource ./rust/stub;
             target = "x86_64-unknown-uefi";
             doCheck = false;
           };
 
+          fatStubCrane = stubCrane.override {
+            extraArgs = {
+              cargoExtraArgs = "--no-default-features --features fat";
```

Comment:
(I wish `crane` provided a proper `‘features` array)

Reply:
That'd be nice